imblearn/ensemble/_weight_boosting.py

```diff
@@ -151,7 +152,7 @@ def fit(self, X, y, sample_weight=None):
         super(RUSBoostClassifier, self).fit(X, y, sample_weight)
         return self
 
-    def _validate_estimator(self, default=DecisionTreeClassifier()):
+    def _validate_estimator(self, default=DecisionTreeClassifier(max_depth=1)):
```

Comment:
I think that you are right. We should actually call the parent class then:
https://github.com/scikit-learn/scikit-learn/blob/7389dba/sklearn/ensemble/weight_boosting.py#L414